src/node_quic_session.cc

```diff
@@ -165,6 +165,540 @@ void QuicSessionConfig::GeneratePreferredAddressToken(ngtcp2_cid* pscid) {
   settings_.preferred_address.cid = *pscid;
 }
 
+// Check required capabilities were not excluded from the OpenSSL build:
+// - OPENSSL_NO_SSL_TRACE excludes SSL_trace()
+// - OPENSSL_NO_STDIO excludes BIO_new_fp()
+// HAVE_SSL_TRACE is available on the internal tcp_wrap binding for the tests.
+#if defined(OPENSSL_NO_SSL_TRACE) || defined(OPENSSL_NO_STDIO)
+# define HAVE_SSL_TRACE 0
+#else
+# define HAVE_SSL_TRACE 1
+#endif
+
+QuicCryptoContext::QuicCryptoContext(
+    QuicSession* session,
+    SecureContext* ctx,
+    ngtcp2_crypto_side side,
+    uint32_t options) :
+    session_(session),
+    side_(side),
+    in_tls_callback_(false),
+    in_key_update_(false),
+    in_ocsp_request_(false),
+    in_client_hello_(false),
```

Comment:
Can you use default initializers in the class definition itself instead for all constants?